### PR TITLE
pwn: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]

File: pages.es/linux/pwn.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# pwn
+
+> Biblioteca de desarrollo de exploits diseñada para la creación rápida de prototipos.
+> Más información: <https://docs.pwntools.com/en/stable/commandline.html>.
+
+- Convierte el código ensamblador dado a `bytes`:
```
Comment (tricantivu) [inline pages.es/linux/pwn.md:6]:
```suggestion
- Convierte código ensamblador a `bytes`:
```

File: pages.es/linux/pwn.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`pwn asm "{{xor edi, edi}}"`
+
+- Crea un patrón cíclico dado un número específico de caracteres:
+
+`pwn cyclic {{número}}`
+
+- Codifica los datos dados en el sistema hexadecimal:
```
Comment (tricantivu) [inline pages.es/linux/pwn.md:14]:
```suggestion
- Codifica datos en el sistema hexadecimal:
```

File: pages.es/linux/pwn.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`pwn hex {{deafbeef}}`
+
+- Decodifica los datos dados en hexadecimal:
```
Comment (tricantivu) [inline pages.es/linux/pwn.md:18]:
```suggestion
- Decodifica datos en hexadecimal:
```

File: pages.es/linux/pwn.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`pwn unhex {{6c4f7645}}`
+
+- Imprime un shellcode de Linux x64 para ejecutar un intérprete de comandos:
+
+`pwn shellcraft {{amd64.linux.sh}}`
+
+- Comprueba la configuración de seguridad binaria para el archivo ELF dado:
```
Comment (tricantivu) [inline pages.es/linux/pwn.md:26]:
```suggestion
- Comprueba la configuración de seguridad binaria del archivo ELF dado:
```

File: pages.es/linux/pwn.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`pwn checksec {{ruta/al/archivo}}`
+
+- Busca actualizaciones de Pwntools:
+
+`pwn update`
+
+- Muestra versión:
```
Comment (tricantivu) [inline pages.es/linux/pwn.md:34]:
```suggestion
- Muestra la versión:
```

File: pages.es/linux/pwn.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# pwn
+
+> Biblioteca de desarrollo de exploits diseñada para la creación rápida de prototipos.
+> Más información: <https://docs.pwntools.com/en/stable/commandline.html>.
+
+- Convierte el código ensamblador dado a `bytes`:
+
+`pwn asm "{{xor edi, edi}}"`
+
+- Crea un patrón cíclico dado un número específico de caracteres:
```
Comment (tricantivu) [inline pages.es/linux/pwn.md:10]:
```suggestion
- Crea un patrón cíclico con un número específico de caracteres:
```

File: pages.es/linux/pwn.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+- Codifica los datos dados en el sistema hexadecimal:
+
+`pwn hex {{deafbeef}}`
+
+- Decodifica los datos dados en hexadecimal:
+
+`pwn unhex {{6c4f7645}}`
+
+- Imprime un shellcode de Linux x64 para ejecutar un intérprete de comandos:
```
Comment (tricantivu) [inline pages.es/linux/pwn.md:22]:
```suggestion
- Imprime código de intérprete de Linux x64 para ejecutar en un intérprete:
```

AFAIK, the word `shellcode` does not exist in Spanish.

**Review by tricantivu — COMMENTED** [review]
I removed the word `dado` and its plural form because I think the descriptions are clearer. Let me know if you disagree.

**Pull request merged**